wet-boew/equalheight/index.json-ld

```diff
@@ -0,0 +1,46 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+		"title": {
+		"en": "Equal height",
+		"fr": "Égalisation des hauteurs"
+	},
+	"description": {
+		"en": "Equalises the height of elements that are on the same baseline",
+		"fr": "Égalise la hauteur d'éléments étant sur la même ligne de base"
+	},
+	"modified": "2024-05-16",
+	"componentName": "equalheight",
+	"status": "stable",
+	"pages": {
+		"examples": [
```

Comment:
Missing one example, the one that use CSS grid and put it first in the JSON array.